contentcuration/contentcuration/frontend/preview/views/Details.vue

```diff
@@ -0,0 +1,365 @@
+<template>
+
+  <div>
+    <LoadingText v-if="loading" />
```

Comment:
From a brief skim, this component looks very similar to the channel details component? What are the differences?

Reply:
The channel details components contains everything up to the channel size, this contains all of the tree details (total resources and on). The reason I implemented it this way was so that it can be easily reused for when we add this summary view to topics

![image](https://user-images.githubusercontent.com/7447496/72107041-687acf80-32e5-11ea-913f-1c6ef2b1182d.png)